app/common/helpers.py

```diff
@@ -0,0 +1,97 @@
+from flask_restful import abort
+
+from app.models import User, BucketList, BucketListItem
+from app import db
+
+
+def register_user(user):
+    """
+    After a successfull registration,
+    it welcomes the user.
+    """
+    return {
```

Comment:
In python, when you create a data structure like this {}, internally python still creates a `dict` to be able to represent that data structure. Therefore, going ahead to create it using the `dict` keyword saves you one step.
```
return dict(
   message='Welcome ' + str(user.username) + ' to the Bucketlist Service.'
)
```